lib/preprocess.js

```diff
@@ -14,6 +14,8 @@ export class Measurement {
     this.participantAddress = pointerize(parseParticipantAddress(m.participant_address))
     this.retrievalResult = pointerize(getRetrievalResult(m))
     this.cid = pointerize(m.cid)
+    this.minerId = pointerize(m.miner_id)
+    this.providerId = pointerize(m.provider_id)
```

Comment:
this field seems not to be used anywhere else

Reply:
We are not using this field yet, but it's already present in the measurements published by spark-publish - see https://github.com/filecoin-station/spark-api/blob/c92260b573db03bbd67d3e809b8ad15a21a71f74/spark-publish/index.js#L31-L32

If you don't mind, I'd like to add it together with `minerId`.

Reply:
On this I can share my recent experience extracting `voyager-{api,evaluate}` from spark. There is quite a few fields we're not using yet, and that was confusing to me, as I assumed if they are in the code they are being used. I thought this was a bug and that they were added by mistake.

If you would like to add them now, could you please add a comment, so the codebase stays consistent?

Reply:
see [1ebaa27](https://github.com/filecoin-station/spark-evaluate/pull/177/commits/1ebaa27e1bd09ff09ffdb94428d99f372d3e4494)